### PR TITLE
Adds readme for 'especialidades'

### PR DESCRIPTION
**Pull request by iteles**
For those not fortunate enough (or without the forethought) to have a single contractor to compile and hand in all the especialidades projects for them, I've compiled this readme to capture what we learnt from doing it ourselves as part of closing off #44.

Hopefully it will either convince people to get someone else to do it for them or help those who have to do it themselves.

Includes link to the list of projects themselves in #36.

### CODE REVIEW

File: town-council-requirements.md [hunk 1]
```diff
@@ -14,8 +14,9 @@ The first step is to get architectural approval for your changes - you should
 make sure you have architects who are able to manage this process for you.
 
 Once this has been approved, you will need to hand in the projects for all of
-your 'specialties' :wrench: :nut_and_bolt: (`especialidades` in Portuguese)
-such as plumbing, electrical, etc. for approval.
+your 'specialties'  🔧 🔩🚿🚽 
+([`especialidades`](https://translate.google.com/#sl=pt&tl=en&text=especialidades) in Portuguese)
```
Comment (iteles) [inline town-council-requirements.md]: Your link is leading to a blank translation. Consider changing to https://translate.google.com/#view=home&op=translate&sl=pt&tl=en&text=especialidades

Comment (nelsonic) [reply town-council-requirements.md]: That's annoying ... 😞 
(_I have it open so will change it now_)

File: town-council-requirements.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+If you don't have AutoCAD (because :money_with_wings:), I found that
+[AutoDesk Viewer](https://viewer.autodesk.com) was a great free tool that
+allowed me to open and check through the `.dwf` plants.
+
+Once that's all good - and especially if you don't want to spend the next few weeks
+trapsing back and forth to the town council with what you hope are correct documents -
+you're going to want to personally check the technical requirements.
+
+I used the [this PDF validator](https://www.pdf-online.com/osa/validate.aspx)
```
Comment (nelsonic) [inline town-council-requirements.md]: not a fan of having `["this"]` in the link title ... doesn't add any value.

File: town-council-requirements.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+handing over all the information contained
+within them and they may as well be in the public realm.  
+**Do not use these for
+documents with personal information**.
+
+
+
+
+### When you go to hand everything in
+Even when all of our documents were finally correct and we went to hand everything
```
Comment (nelsonic) [inline town-council-requirements.md]: this is a _super_ long sentence ...

**Pull request merged**